InterviewFlashCards/JsonFileReader.swift

```diff
@@ -22,14 +22,23 @@ struct JsonFileReader: DataGenerator {
         guard let path = fileName.resourcePath,
             let jsonData = try? Data(contentsOf: path,
                                      options: .mappedIfSafe),
-            let json = (try? JSONSerialization
-                .jsonObject(with: jsonData as Data,
-                            options: .allowFragments)) as? [String: Any],
-            let data = json["data"] as? JSON else {
+            let json = json(from: jsonData),
+            let data = json["data"] as? [JSON] else {
                 completion([])
```

Comment:
@mesbekmek a little confused, because I don't see where json is an array of dictionaries? the return type for the function is a dictionary, and `return json as? [String: Any]` is a dictionary too so i'm not following :/ 

also previously it was `public typealias JSON = [[String: Any]]` which is the array of dictionaries, but in the newest commit `JSON` becomes a dictionary?